src/app/screens/ReceiveInvoice/index.tsx

```diff
@@ -264,7 +264,11 @@ function ReceiveInvoice() {
                   fullWidth
                   primary
                   loading={loadingInvoice}
-                  disabled={loadingInvoice}
+                  disabled={
+                    loadingInvoice ||
+                    !formData.amount ||
+                    formData.amount === "0"
```

Comment:
@rolznz how many of the connectors support amount less invoice, I think lndhub supports that. even if it not supports we throw the error. so probably we remove the third condtion. rest looks good

Reply:
I don't think a user should have to manually specify "0" for an amountless invoice. I see Lndhub works fine right now - I paid an amountless invoice created in Alby using a different wallet. (I know the Alby extension cannot pay amountless invoices yet).

So maybe we should actually not merge this PR and not change anything.

@bitcoinuser do you remember why you made this issue?

Reply:
> @bitcoinuser do you remember why you made this issue?

To be consistent with others windows of Alby, like the send one, that the button are disabled until we insert an information in the field.

Reply:
> I don't think a user should have to manually specify "0" for an amountless invoice.

Why not show the qrcode on receive window for amountless invoice like what we have for custodial alby wallet? In the case of Alby account show lightning address qrcode, but for non custodial wallets, could show a amountless qrcode, no?

Reply:
I don't think we can always rely on a QR code (unless there is a copy button as well). But I think it's unrelated to this issue. Like Pavan said I would just remove this check and not allow 0-amount invoices and then this PR looks ok to me :rocket:

Reply:
@bitcoinuser @rolznz @pavanjoshi914 Thank you for your valuable feedback. I have removed the third check.